microsoft/PowerToys
# Mouse Utils - Inclusive Mouse

## Summary of the Pull Request

**What is this about:**
Adds a new PowerToy called "Inclusive Mouse", under "Mouse Utilities".
This PowerToys draws a crosshair around the mouse pointer in its current screen, target at a set of users with specific accessibility needs.
This PowerToy has been designed originally by the accessibility team.
Default activation key is Ctrl+Alt+P.

![image](https://user-images.githubusercontent.com/26118718/150056682-6dbb2515-67ea-4184-8930-729811c5caef.png)

**What is included in the PR:** 
- The code for the PowerToys.
- Settings page and Oobe description line.
- Logging and telemetry.

![image](https://user-images.githubusercontent.com/26118718/150056747-49e25be2-9eb7-41ed-998e-eed15e3bf27f.png)

**How does someone test / validate:** 
Start PowerToys with the module enabled and press "Ctrl"+"Alt"+"P". Move the mouse around and check it has a crosshair following it "Ctrl"+"Alt"+"P" to toggle it off again.

## Quality Checklist

- [x] **Linked issue:** #15632

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
